src/lib/generators/code-snippet-generators.spec.ts

```diff
@@ -1,19 +1,96 @@
-import { generateInterfaceDefinition, generateSvgConstant, generateTypeDefinition } from './generators';
+import {
+  generateInterfaceDefinition,
+  generateSvgConstant,
+  generateTypeDefinition,
+  generateVariableName
+} from './code-snippet-generators';
+import { MultiFileConvertionOptions, SingleFileConvertionOptions } from '../options/convertion-options';
+import { SvgDefinition } from '../converters/shared.converter';
 
 describe('Generators', () => {
-  it('should return the correct interface definition', () => {
-    const interfaceName = 'TestIcons';
-    const typeName = 'myAwesomeIcons';
-    const expectedDefinition = `export interface ${interfaceName}{
-        name: ${typeName};
+  describe('generateInterfaceDefinition', () => {
+    it('should return the correct interface definition when passing generateType', () => {
+      const options = { generateType: true, typeName: 'AppIcons', interfaceName: 'IconInterface' } as
+        | MultiFileConvertionOptions
```

Comment:
I think the | is not needed

Reply:
correct, I just copied the type from the signature 😄